content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/scrap_inventory.rst

```diff
@@ -2,93 +2,129 @@
-PROCESS` button on the :guilabel:`Receipts`, :guilabel:`Internal Transfers`, or :guilabel:`Delivery
-Orders` card, depending on the type of operation the product is being scrapped from.
+Once ready, click :guilabel:`Validate` to complete the new |SP|. Once validated, a
+:guilabel:`Product Moves` smart button appears at the top of the form. Click the smart button to
+view the details of the scrap operation.
 
-.. note::
-   For the :guilabel:`Internal Transfers` card to appear on the :menuselection:`Overview` page of
-   the :menuselection:`Inventory` app, the :guilabel:`Storage Locations` setting must be enabled. To
-   do so, navigate to :menuselection:`Configuration --> Settings`, then enable the checkbox next to
-   :guilabel:`Storage Locations` under the :guilabel:`Warehouse` heading.
+.. image:: scrap_inventory/scrap-inventory-product-moves-button.png
+   :align: center
+   :alt: Product Moves smart button on new scrap order form.
+
+.. tip::
+   To view the all-time total quantities of scrapped items, navigate to :menuselection:`Inventory
+   app --> Configuration --> Locations`. Click the :guilabel:`x (remove)` button on the
+   :guilabel:`Internal` filter in the :guilabel:`Search...` bar, to display virtual locations.
+
+   Select the :guilabel:`Virtual Locations/Scrap` location. From the :guilabel:`Scrap` location's
+   form, click the :guilabel:`Current Stock` smart button, at the top of the form.
+
+   A list of all scrapped products, and their quantities, is displayed.
+
+   .. image:: scrap_inventory/scrap-inventory-current-stock.png
+      :align: center
+      :alt: Current Stock list of all scrapped products in virtual scrap location.
+
+Scrap from an existing operation
+================================
+
+Scrap orders (SPs) can *also* be created from existing operations, such as receipts, delivery
+orders, and internal transfers, before they are entered into, or removed from, stock for an
+operation.
+
+To scrap a product during an operation, navigate to the :menuselection:`Inventory app`. From the
+:guilabel:`Inventory Overview`, click the :guilabel:`# To Process` button on an operation's task
+card (i.e. the :guilabel:`Receipts` task card).
+
+.. image:: scrap_inventory/scrap-inventory-receipts-task-card.png
+   :align: center
+   :alt: # To Process button on Receipts task card on Inventory Overview page.
 
-Alternatively, a list of all delivery orders, receipts, and transfers can be viewed by navigating to
-:menuselection:`Inventory --> Operations --> Transfers`.
+Then, select an operation to process from the resulting list of existing orders. Doing so opens that
+operation's form.
 
-Next, open a delivery order, receipt, or transfer from the corresponding page by clicking on it. A
-:guilabel:`Scrap` button appears at the top of the page. Click it to open the :guilabel:`Scrap`
-pop-up window.
+Click the :guilabel:`⚙️ Actions (gear)` icon, and select :guilabel:`Scrap` from the resulting
```

Comment:
Since the text "Actions" is in a tooltip, I dont think we should include it in the icon guilabel, maybe state "from the actions drop-down menu"?
```suggestion
Click the :guilabel:`⚙️ (gear)` icon, and select :guilabel:`Scrap` from the resulting
```
Also feel free to use the FA icon role for this moving forward :)
```suggestion
Click the :icon:`fa-cog` :guilabel:`(cog)` icon, and select :guilabel:`Scrap` from the resulting
```
This does bring up several points to discuss with the team regarding whether or not to use the icon class name or it's given tool-tip name in the :guilabel:, and furthermore, is "gear" a better label than "cog"? I'm going to use this as an example to start this discussion :)

<img width="834" alt="Screenshot 2024-05-09 at 11 46 08 AM" src="https://github.com/odoo/documentation/assets/36018073/0218bd2f-0909-4630-a165-563aa0244bab">

Reply:
Decided to go with the FA icon and the "cog" label for this one - I think staying consistent with the FA icon role is good practice because not everything in Odoo has a useful tooltip description ... but I agree this is a good discussion to have! Suggestion implemented ✅